aaad/utils.py

```diff
@@ -44,3 +44,12 @@ def getAllowedNamespacePatterns(act_as, permissions):
 
     return allowed_namespace_patterns
 
+def merge_dicts(dict1, dict2):
```

Comment:
@markg-moz that would be wrong here, as it will override the value and not merge.